ui/next.config.js

```diff
@@ -1,6 +1,16 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-    reactStrictMode: true
+    reactStrictMode: true,
+
+    async redirects() {
```

Comment:
This is now showing a blank page for 1-2 seconds when pressing the Search button from the Header.
Not sure if using a middleware for this redirect has the same issue. But for now, we can make that Search button from the Header to redirect the user straight to '/search/publications' instead of '/search' and avoid the redirect when clicking on it.